theme/blog/index.html.ep

```diff
@@ -1,9 +1,22 @@
-<p>This is the blog</p>
+<h1>News</h1>
```

Comment:
I imagined the blog as, well, a blog. That is, with the index containing all posts, but perhaps the sidebar showing only the news (e.g. blog post with the tag `news`).

In the spirit of following the smallest incremental changes which already add value, we can start with "news-only" as this header suggests.

Reply:
Ok. Seems like a good plan. I've left this heading in place.